src/protocols/ssh/input.c

```diff
@@ -58,7 +174,7 @@ int guac_ssh_user_key_handler(guac_user* user, int keysym, int pressed) {
 
     /* Report key state within recording */
     if (ssh_client->recording != NULL)
-        guac_recording_report_key(ssh_client->recording,
+        guac_common_recording_report_key(ssh_client->recording,
```

Comment:
This should remain `guac_recording_report_key()`. There is no `guac_common_recording_report_key()` function.